Makefile

```diff
@@ -1,7 +1,12 @@
-# Variables are documented in hack/build.sh.
+# Variables are documented in common/build.sh.
 BASE_IMAGE_NAME = postgresql
 VERSIONS = 9.2 9.4 9.5
 OPENSHIFT_NAMESPACES = 9.2
 
 # Include common Makefile code.
-include hack/common.mk
+-include common/common.mk
+git_submodule_warning:
+	git submodule update --init
+
+test: git_submodule_warning
+	$(MAKE) check $(MAKEFLAGS)
```

Comment:
nono, the second commit is there just to make it work without fixing CI ... if this gets merged, we should have CI doing the submodule-init explicitly.